### ISSUE TITLE
Is it possible to find the date of a change?

### ISSUE THREAD

**Issue by icefoxen** [Byron/crates-index-diff-rs#2]
Unless you go to poke the `crates.io` HTTP API, the only way to find the publication date of a particular version of a crate is to grovel through the index git repository.  I'm going to need to do this eventually, but since this project already does a bit of groveling through the index git repository, would you be interested if I just contributed a PR to add this functionality?

**Comment by Byron** [comment]
Yes, this crate is happily accepting PRs :), thanks for offering to give it
a try.

**Comment by Byron** [comment]
Just now I implemented 'ordered changes', which steps through the history one by one. With that one would have enough information to attach a precise date to each change.
